File: frontend/src/constants/urls.js
export const BaseUrl = '/api'
const advertisement = '/ads'
const cars = '/cars'

export const urls = {
    advertisement :{
        get:`${advertisement}/get`,
        create:`${advertisement}/create`,
        image:`${advertisement}/image`,
        update:`${advertisement}/update`

    },
    cars:{
        get_brands:`${cars}/get/brand`
    }

}

File: backend/apps/advertisement/urls.py
from django.urls import path

from apps.advertisement.views import CreateAdvertisementApiView, CarAddImageView, ListAdvertisementApiView, \
    UpdateAdvertisementApiView, GetOneAdvertisementApiView

urlpatterns = [
    path('/create', CreateAdvertisementApiView.as_view()),
    path('/image/<int:pk>', CarAddImageView.as_view()),
    path('/get', ListAdvertisementApiView.as_view()),
    path('/update/<int:pk>', UpdateAdvertisementApiView.as_view()),
    path('/get/<int:pk>',GetOneAdvertisementApiView.as_view())
]
